README.md

```diff
@@ -1,2 +1,25 @@
-# HW01
-Edit the readme
+# HW01. Ricardo Muñiz's Bio
+
+## Introduction
+
+Hi, my name is **Ricardo Muñiz Trejo**, I'm **27** years old, and I'm **Mexican**. I was admitted to the **Ecology & Evolution** Ph.D. program, and I'm very excited about it!
+
+I've lived in Mexico City my whole life, so moving to Chicago will be a novel experience. Let's just hope I can go there, given the pandemics. Speaking of which, here's a picture of me *rocking* a mask \#BeSafe \#WearAMask.
```

Comment:
I visited Mexico City a few years ago and loved it!

Reply:
I'm glad you loved it! There is a lot to see, even when I lived my entire life here, I can always discover new places.